vendoring.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Not sure on the name for this file. I like the `editor-` from the previous incarnation. How about `editor-components.json`? It keeps it specific to us, and (to me) does not imply a general dependency file. I may be over thinking this